crates/routes/Cargo.toml

```diff
@@ -34,4 +34,5 @@ once_cell = { workspace = true }
 tracing = { workspace = true }
 tokio = { workspace = true }
 urlencoding = { workspace = true }
+serde_with.workspace = true
```

Comment:
Should this specify workspace the same way the other dependencies do?

Reply:
Yes that would be cleaner.